src/ui/uiLayer.js

```diff
@@ -20,8 +20,14 @@ geo.gui.uiLayer = function (arg) {
   geo.layer.call(this, arg);
 
   var m_this = this,
+      s_init = this._init,
       s_exit = this._exit;
 
+  this._init = function () {
+    s_init();
+    m_this.node().css('position', 'relative');
```

Comment:
Yes, we will want to manage most css properties using classes like `.geo-map`, `.geo-layer.geo-ui-layer`, `.geo-widget`, etc.